index.bs

```diff
@@ -404,10 +406,10 @@ spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
 
     1. [=Assert=]: |element| has a [=guest browsing context=].
 
-    1. If |element| has no {{HTMLPortalElement/src}} attribute specified, or its value is the empty string,
+    1. If |element| has no "`src`" attribute specified, or its value is the empty string,
```

Comment:
You want `<{portal/src}>`. This means you'll need a definition for the `src=""` content attribute (`<dfn element-attr for="portal">src</dfn>`). See e.g. https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-iframe-src

This also touches briefly on #71, in that the sentence

> The attribute, if present, must be a valid non-empty URL potentially surrounded by spaces. 

for iframe is an authoring conformance requirement, not an implementer requirement.

Reply:
Man, I Ctrl+F-ed through tabatkins.github.io/bikeshed/ looking for how "element-attr" was spelled and couldn't figure it out. Thanks. :)

Reply:
Done (though I've left #71 as something to do in a later PR, if that's alright).